src/internal/packager/images/pull.go

```diff
@@ -144,6 +144,7 @@ func Pull(ctx context.Context, cfg PullConfig) (map[transform.Image]v1.Image, er
 
 					// TODO(mkcp): Remove message on logger release
 					message.Warnf("Falling back to local 'docker', failed to find the manifest on a remote: %s", err.Error())
+					l.Warn("Falling back to local 'docker', failed to find the manifest on a remote", "error", err.Error())
```

Comment:
Unrelated, but noticed this the other day. IMO needed otherwise it's hard to tell why you're pull failed (this happened to me the other day when my PAT expired)